src/common/libsubprocess/remote.c

```diff
@@ -587,7 +588,7 @@ static int remote_output (flux_subprocess_t *p, flux_future_t *f,
 
         if (tmp != len) {
             flux_log_error (p->h, "channel buffer error: rank = %d pid = %d, stream = %s, len = %d",
-                     rank, pid, stream, len);
+                     rank, pid, stream, (int)len);
```

Comment:
instead of casting, perhaps should just use ```%zu``` (i think) specifier?

Reply:
Ok, fixed and forced a push.